### DISCUSSION TITLE
Is it possible to make this script faster?

### DISCUSSION THREAD

**Discussion by pugrex** [jef/streetmerchant#1918] [Help]
Im using it for one store only, and is it possible for it to check stocks faster? It takes almost one minute for it to check the same item, im only using 3070 and 3080. In one minute its already been sold. Can i make it faster by any chance?

**Comment by Lewis-Bright** [comment]
There's a few settings you can try changing in the `dotenv` file, such as `PAGE_BACKOFF_MIN` and `PAGE_SLEEP_MIN` that might help speed it up (more info -> https://jef.codes/streetmerchant/reference/application/).

If you're finding it's taking a full minute to loop through all the cards on the store then there's not really much that can be done, as I think the main time taken is just to get the response from the site. You can try only showing models/brands that are relevant if you can think of a way to restrict that (e.g. not showing the famously expensive super OC versions)
